### DISCUSSION TITLE
How that html slideshow was done in R ?

### DISCUSSION THREAD

**Discussion by Andrzej-Andrzej** [quarto-dev/quarto-cli#5720] [Q&A] [answered]
### Description

I would like to make a similar presentation:
[https://ardata-fr.github.io/flextable-whyr/#1](https://ardata-fr.github.io/flextable-whyr/#1)
What tools should I use ? I would like to have code, output, tables, pictures in it, paginations, etc.
When I press down arrow, pages should move forward. How do I start with it ? So far I used PowerPoint only for slides, I would like to move to the next level and create such a slideshow like that I provided in the link above. Please advise.

**Comment by mcanouil** [comment, marked as answer]
The slides are made with `xaringan` (R package using Remark.js).

Quarto/Pandoc doesn't use Remark.js, but uses Reveal.js.

Please consider looking at Quarto documentation website <https://quarto.org/> before asking a question. In this case, the tutorials "Get Started" and the guides.

**Reply by Andrzej-Andrzej** [reply]
Thank you, 


> The slides are made with `xaringan` (R package using Remark.js).
> 
May I ask how do you recognize this ?  Is it possible to find a code for that presentation somewhere ? I will read guides because I want to prepare similar slideshow with Quarto.

**Reply by mcanouil** [reply]
Because I know `xaringan` / Remark.js (html source code should also have the information).

You'll have to figure where the source code is yourself:
- reach out to the author.
- look for a link in the slides.
- look at GitHub user/organisation for the corresponding repository.

Finally, as I said, `xaringan` has nothing to do with Quarto and uses different syntax.

**Reply by cderv** [reply]
> Is it possible to find a code for that presentation somewhere ?

Yes on the last slide: https://ardata-fr.github.io/flextable-whyr/#39
